### DISCUSSION TITLE
Amazon Api didn't respond with Okay, see exception for more details

### DISCUSSION THREAD

**Discussion by LouGibson** [abuzuhri/Amazon-SP-API-CSharp#440] [General]
Hello,

I'm seeing this error intermittently when calling AmazonConnection.Orders.GetOrders. Could you advise which property contains the "exception" where I can see more details.

Many thanks,
Lou

**Comment by ProNotion** [comment]
@LouGibson In case it helps - I think I was getting that in the past when passing an invalid date or date range for the orders.

**Reply by LouGibson** [reply]
Thanks @ProNotion, I was using an older version of the Amazon-SP-API which didn't support the IsDebugMode property. After implementing this I can see that the Amazon service is returning “We encountered an internal error. Please try again”. Hopefully they'll get this fixed soon.

Regards,
Lou
